### ISSUE TITLE
Increase maxInclusive value of number-level

### ISSUE THREAD

**Issue by hhpmusic** [w3c/musicxml#297]
In [This part](http://usermanuals.musicxml.com/MusicXML/MusicXML.htm#ST-MusicXML-number-level.htm) of Musicxml Documentation, the maxInclusive value of number-level is 6. But in lots of circumstances I found there are slurs exceeding this limit. For example, in a very complicated piano concerto, when lots of cross-staff slurs are met, and when in a part created with lots of sub-staves and every staff has its own slur. In this case, the export will be incomplete, and all subsequent slurs will be lost after the exhaust of all 6 levels. I suggest to increase the level to at least 15 or 16 (I found a Sibelius file of Ligeti's Atmosphere, in which each violin group is created as a single part with 14 staves!).

Haipeng

**Comment by mdgood** [comment]
@hhpmusic Thank you for raising this issue Haipeng. In MusicXML, different numbers for slurs are only needed when the slurs overlap in document order. We have done a lot of piano music examples and have not yet found a need for more than 6 levels. If you have a specific example could you please post that here?

Using a single 14-staff part for Ligeti's Atmospheres would indeed exceed the limit if each staff has a slur over the same barline. But that seems to me to be a symptom of a different MusicXML issue with representation of condensed scores, more than a slur numbering issue.

**Comment by hhpmusic** [comment]
Ok, but I'll contact you privately since the file is not open.

**Comment by mdgood** [comment]
Thank you @hhpmusic. I have received the files and plan to look at these soon.

**Comment by mdgood** [comment]
Thank you again @hhpmusic for sending on those files. I wanted to look at this issue before starting work on #8, as the two seem interconnected.

In the piano part example, I think the problem is in the export software not correctly recognizing the cross-staff slurs, and then running out of levels. When I look at the place in the music where the export warns this is happening, it should be working correctly from the MusicXML format perspective.

As mentioned before, I think the Atmosphères issue is something different, where MusicXML is not doing a great job handling this type of changing large-scale divisi within a single part.

In summary, I think we can leave the number-level restriction in place for now. I do think for the cross-staff arpeggio issue #281 we do want to increase the level. However we can do that by creating a separate type that increasing the number of simultaneous cross-staff arpeggios without changing the number of levels for other elements.

I will plan to close this issue. Would you like to write up the Atmosphères issue separately, or would you prefer that I do that?

**Comment by mscuthbert** [comment]
Atmosphères is I believe what is on Don Byrd's extremes of CMN for number of divisis or staves, etc.

**Comment by mdgood** [comment]
Thanks @mscuthbert! Yes, Atmosphères is listed there as the record holder for "most simultaneous notated staves" and "most simultaneous notated voices." Alas there is no category for "most simultaneous divisi voices in a single part" but it seems reasonable to expect this would be pretty high up there.

If we increase the maxInclusive level to 16 as @hhpmusic suggests, I think that would be enough to handle this score. We could address some of the documentation issues as part of #277. Increasing the level to 16 also means that arpeggios would not have a separate number-level as part of #281.

We could document this change to indicate that most music can expect a maximum value of 6, but that the extended range is available for scores with large-scale divisi across more than 6 staves.

Instead of closing this, I am leaning towards adding it into scope for 4.0. It seems to fit in well with the other two issues that are currently in the 4.0 milestone. Like #305, it also lets MusicXML support some scores that MusicXML 3.1 cannot represent fully accurately in a standard way.
